src/components/Dashboard/FilingHistoryList/filings/DefaultFiling.vue

```diff
@@ -0,0 +1,20 @@
+<template>
```

Comment:
The is the default filing component if we don't need a custom component for a filing type.